src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogHttpClient.java

```diff
@@ -54,7 +54,9 @@ of this software and associated documentation files (the "Software"), to deal
  */
 public class DatadogHttpClient implements DatadogClient {
 
-    private static DatadogClient instance;
+    private static DatadogClient instance = null;
+    private static boolean failedLastValidation = false;
```

Comment:
Let's have a comment to tell why it's useful.

Reply:
Sure!